.ci/scripts/cslc_s1/test_int_cslc_s1.sh

```diff
@@ -184,7 +184,7 @@ if [ $overall_status -eq 0 ]; then
                                 -v "${expected_dir}":/exp:ro \
                                 --entrypoint /home/compass_user/miniconda3/envs/COMPASS/bin/python3 \
                                 ${PGE_IMAGE}:"${PGE_TAG}" \
-                                /working/validate_product.py \
+                                "${SCRIPT_DIR}"/../cslc/cslc_s1_compare.py \
```

Comment:
Since this script is now part of the repository, we do not need to invoke it as part of a `docker run` command.

Basically, lines 181-190 can be replaced with something like the following:

`compare_out=$("${SCRIPT_DIR}"/../cslc_s1/cslc_s1_compare.py --ref-product ${ref_product} --sec-product ${sec_product} -p CSLC 2>&1) || compare_exit_status=$?`

Usage of `$docker_out` should then be replaced with `$compare_out`. 
This also needs to be applied to lines 220-229 below.